Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -116,6 +116,12 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UFUNCTION(BlueprintCallable, BlueprintPure, Category="C++")
 	UPSSpotComponent* GetCurrentSpot() const;
 
+	/** Returns Progression Star Dynamic Material by state
+	 * Each state has own instance Dynamic Material Instance 
+	 * @param StarState a star state (Locked, Unlocked, Partial) */
+	UFUNCTION(BlueprintCallable, BlueprintPure, Category="C++")
+	UMaterialInstanceDynamic* GetStarProgressionDynamicMaterial(EPSStarActorState StarState);
```

Comment:
`class` specifier is missing before pointer, which will cause the warning

Reply:
done https://github.com/moinm3uw/ProgressionSystem/pull/55/commits/91681a68fcd68518f1b31f23ce1908671a5f0da2